bots/mlbridge/src/main/java/org/openjdk/skara/bots/mlbridge/ArchiveMessages.java

```diff
@@ -315,7 +315,7 @@ static String composeConversationFooter(PullRequest pr, URI issueProject, String
                 "Commit messages:\n" +
```

Comment:
```
// When changing this, ensure that the PR pattern in the notifier still matches
```
Have you verified this?

Reply:
I tried to find the related PR pattern again this morning, but unfortunately I still couldn't find it. I'm still confused about where to look for it. AFAIK, it's just the footer of the email.  Is there some place in the code where we would use this email and and parse it? I also tried resetting my head to the commit where the comment was added, but I couldn't find anything related to the notifier.